dev-games/openscenegraph-openmw/openscenegraph-openmw-3.4_p20200425.ebuild

```diff
@@ -0,0 +1,146 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0"
+inherit cmake flag-o-matic vcs-snapshot wxwidgets
```

Comment:
Is `vcs-snapshot` required here? It seems to be from Github. You can just define `S` globally.